docusaurus/docs/reference/30-configuration/conventions.md

```diff
@@ -57,8 +57,7 @@ overview useful to understand the "default" assumptions. The `identity` section
 - `server_key` - (optional) A string in the format of `<engine>:<value>` that defines a private key for `server_cert`,
   if not
   defined `key` is used if `server_cert` is defined
-- `ca` - (optional) A string in the format of `<engine>:<value>` that defines x509 certificate chain used to define
-  trusted CAs
+- `ca` - (optional) A string in the format of `<engine>:<value>` that defines a readable file containing a trust bundle of CA certificates as PEM. This is not optional when configuring the controller's router control plane identity. In that case, this bundle must include the CA certificates for all server certificates that are presented by the controller, e.g., control plane, web binding for edge client API. This trust bundle is distinct from the PKCS7 trust store that is delivered to SDKs via the `/.well-known/est/cacerts` endpoint in the edge client API.
```

Comment:
way too many words. it's also not any more clear to me. It says optional/not optional and mentions the controller.

Leave this as it is and instead I'd rather see this sort of thing the individual configuration sections for each config

https://docs.openziti.io/docs/reference/configuration/controller/#identity (controller.md)
https://docs.openziti.io/docs/reference/configuration/router#identity (router.md)

revert this and update those sections accordingly

Reply:
@dovholuknf resolving this because it's out of date after I rewrote it.